LayoutTests/inspector/canvas/recording-2d-full-expected.txt

```diff
@@ -305,19 +305,6 @@ frames:
         3: (anonymous function)
         4: executeFrameFunction
   15: (duration)
-    0: drawImageFromRect([object HTMLImageElement], 1, 2, 3, 4, 5, 6, 7, 8, "")
```

Comment:
maybe you need to remove the calls in recording-2d.js ?

Reply:
Done. :-)